src/index.ts

```diff
@@ -0,0 +1,69 @@
+import {lintProject} from "./linter/linter.js";
+import type {LintResult} from "./linter/LinterContext.js";
+
+export type {LintResult} from "./linter/LinterContext.js";
+export type {UI5LintConfigType} from "./utils/ConfigManager.js";
```

Comment:
is this type relevant / necessary for outside consumers?

Reply:
Thanks! I have forgotten to remove it